### PR TITLE
adding new prometheus test

### PR DESCRIPTION
**Pull request by cniackz**
This tag is linked to: https://github.com/miniohq/engineering/issues/356
Where the idea is to create new test for operator and make sure that after tenant is deployed
prometheus is reachable.
Fixes miniohq/engineering#356

### CODE REVIEW

File: testing/check-prometheus.sh [hunk 1]
```diff
@@ -0,0 +1,66 @@
+#!/usr/bin/env bash
```
Comment (dvaldivia) [inline testing/check-prometheus.sh:1]: this is not getting used by any `github action`

**Comment by dvaldivia** [comment]
the new bash file is not executable 
Screen Shot 2022-01-11 at 11 25 48 AM

**Comment by cniackz** [comment]
Ok guys, this is finally working on GitHub Actions.
Please help me to review this new test added for Prometheus.
Tomorrow, I plan to organize it better and split it on functions as Harsha did it.
But the funtionality is going to remain same.
One thing I have noticed, is that kubectl wait will not wait on a resource if it is not present.
Hence I decided to add while and break it until the pod is present and ready, maybe
we can make some core functions around this functionality, because might be
useful for future and present tests. Anyway, the last comment is the fact that our
port-forward was failing and I found that ufw helped me to get port open in Ubuntu VM.
Feel free to put any comment and tomorrow I will be addressing the changes.

**Review by dvaldivia — APPROVED** [review]
LGTM.

**Review by Alevsk — APPROVED** [review]
👍

**Pull request merged**